### DISCUSSION TITLE
Open Props svg logo

### DISCUSSION THREAD

**Discussion by b-d-m-p** [argyleink/open-props#504] [Show and tell]
The logo on the site is an svg, but the styles are applied with css, so it's not easily portable, say if you wanted to use it in your own site, so I styled the one from the site, in Figma, so others can use it easily. 😃
![open-props logo](https://github.com/argyleink/open-props/assets/1300390/cc85760f-bc09-4431-a501-294b13839933)

**Comment by argyleink** [comment]
Rad!
